src/components/layout/FooterSection.vue

```diff
@@ -50,13 +50,33 @@
         <img src="../../assets/iconVenture.svg">
       </a>
     </div>
+    <div
+      v-if="!isLoggedIn"
+      class="ww-login-footer"
```

Comment:
What means `ww`? If it is a web wallet, then can't it be used by the Cordova version?

Reply:
Yes, this is the mobile version of the button for login with web wallet.